api/middleware/applicationMiddleware.js

```diff
@@ -29,8 +28,8 @@ const cacheSignUpData = async (req, res, next) => {
     tech_stack: formData.tech_stack,
     job_help: formData.job_help,
     pair_programming: formData.pair_programming,
-    other_info: formData.other_info,
     referred_by: formData.referred_by,
+    other_info: formData.other_info,
```

Comment:
The changes in this file are resultant of a git pull after new branches were merged.